src/main/java/presenters/helpFrame.java

```diff
@@ -0,0 +1,127 @@
+package presenters;
+
+import javax.swing.*;
+import java.awt.*;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class helpFrame extends JFrame implements ActionListener {
+    JButton button_submit;
+    JButton open_fridge;
+    JButton search;
+    JTextField ingredient_input;
+
+    List<String> fridgeFood = new ArrayList<>();
+
+    JComboBox<String> mealType;
+    JComboBox<String> time;
+    JComboBox<String> calories;
+
+    helpFrame(){
+        //creates the searchpage
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        this.setSize(1280,720);
+
+        //creates the button for submitting the ingredients
+        button_submit = new JButton("Submit");
+        button_submit.addActionListener(this);
+        button_submit.setBounds(560, 100, 280, 50);
+
+        //creates the textfield for user to input
+        ingredient_input = new JTextField();
+        ingredient_input.setPreferredSize(new Dimension(300, 25));
+        ingredient_input.setBounds(60, 100, 500,50);
+
+        //creates the test"Put your ingredients here:" and set it to appropriate place and font
+        JLabel label1 = new JLabel();
+        label1.setText("Put your ingredients here: (One at a time)");
+        label1.setBounds(60,50,800,50);
+        label1.setFont(new Font(null, Font.PLAIN, 25));
+
+        //creates panel1 and add label1, submit button, and textfield to it
+        JPanel panel1 = new JPanel();
+        panel1.setBounds(0, 0, 840, 240);
+        panel1.add(label1);
+        panel1.add(button_submit);
+        panel1.add(ingredient_input);
+        panel1.setLayout(null);
+
+        //create open_fridge button which pops up a messagebox of what ingredients fridge has.
+        open_fridge = new JButton("open your fridge");
+        open_fridge.setBounds(300, 80, 240, 80);
+        open_fridge.addActionListener(this);
+
+        //sets up panel2 and add open_fridge button to it
+        JPanel panel2 = new JPanel();
+        panel2.setBounds(0, 240, 840, 480);
+        panel2.setLayout(null);
+        panel2.add(open_fridge);
+
+        //sets up panel3
+        JPanel panel3 = new JPanel();
+        panel3.setBounds(840, 0, 440, 720);
+        panel3.setLayout(null);
+
+        //creates filter comboBoxes and add it to panel3
+        String[] mealStr = {"no meal filter", "breakfast", "brunch", "lunch/dinner", "snack", "teatime"};
+        mealType = new JComboBox<>(mealStr);
+        mealType.addActionListener(e -> System.out.println(mealType.getSelectedItem()));
+        mealType.setBounds(50, 50,300,40);
```

Comment:
Why are we printing out to the console?

Reply:
This actually needs to pass to controllers and apply the filter, but since we didn't put anything together, I just let it print to the console, it will need to be changed.